embassy-sync/src/channel/priority.rs

```diff
@@ -0,0 +1,614 @@
+//! A queue for sending values between asynchronous tasks.
+//!
+//! Similar to a [`Channel`](crate::channel::Channel), however [`PriorityChannel`] sifts higher priority items to the front of the queue.
+//! Priority is determined by the `Ord` trait. Priority behavior is determined by the [`Kind`](heapless::binary_heap::Kind) parameter of the channel.
+
+use core::cell::RefCell;
+use core::future::Future;
+use core::pin::Pin;
+use core::task::{Context, Poll};
+
+use heapless::binary_heap::Kind;
```

Comment:
could you `pub use` everything needed so the user can use this without imprting `heapless` directly? I think it's `Kind`, `Min`, `Max`.